frontend/Dockerfile

```diff
@@ -12,15 +12,34 @@ FROM base AS shared
 
 WORKDIR /opt
 
-COPY shared/ ./shared
-RUN yarn workspace vaa-shared install --frozen-lockfile
-RUN yarn workspace vaa-shared build
+# Install all shared modules used by the frontend
+# If adding more, remember to copy them below in both places
+COPY vaa-app-shared/ ./vaa-app-shared
```

Comment:
Great! I'll fix this too in #599.

Reply:
Uh, our ancient Yarn 1 throws with this because the `shared-config` package doesn't have a `build` script. This will work with Yarn 4, though, when we upgrade. See #622.